addons/account_peppol/wizard/peppol_registration_views.xml

```diff
@@ -6,26 +6,39 @@
         <field name="group_ids" eval="[Command.link(ref('account.group_account_manager'))]"/>
         <field name="arch" type="xml">
             <form class="form_peppol_registration">
-                <field name="company_id" invisible="1"/>
                 <sheet>
-                    <div class="m-0" name="warnings" invisible="not peppol_warnings or account_peppol_proxy_state != 'not_registered'">
+                    <div class="m-0" name="warnings" invisible="not peppol_warnings or wizard_peppol_proxy_state != 'not_registered'">
                         <field name="peppol_warnings" class="o_field_html" widget="actionable_errors"/>
                     </div>
 
-                    <div name="peppol_registration" invisible="account_peppol_proxy_state not in ('not_registered', 'in_verification')">
```

Comment:
Good catch, "in_verification" is indeed removed in master. Thanks for the fix :ok_hand: